src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private int capacity;
+
+    private String description;
+
+    public CinemaHall() {
```

Comment:
There's a default constructor defined here, which is not needed since there are no other constructors with parameters that would make the provision of a default constructor necessary for the Java compiler. You can safely remove this default constructor to clean up the code.